Cakefile

```diff
@@ -107,9 +107,9 @@ task "html", "Build HTML", (options) ->
   scripts = []
 
   if options.test
-    scripts.push "http://test.e17.dk/getnotaauthtoken"
+    scripts.push "http://nota.dk/getnotaauthtoken"
```

Comment:
Maybe? I don't know. I just remember Michael saying that there was no test version at that point. More importantly: Do the test env have its own seperate session/login system or does it just reuse the live one?